### **Awesome LifeScience**

A collection of awesome things regarding the LifeScience.

## Contents
- [Tools](#tools)
- [Pipelines](#pipelines)
- [Datasets](#datasets)
- [Visualization](#visualization)

## Tools
- Genomics
    - [fastp](#https://github.com/OpenGene/fastp)
    - [gatk](#gatk)
- [Protein](#protein)
- [Cell](#cell)

## Pipelines
- Genomics
    - [Metagenome](#metagenome)
    - [RNA-seq](#rna-seq)
    - [WGS](#wgs)
    - [WES](#wes)

## Datasets

## Visualization
- Genomics
    - General
        - [Biodalliance](http://www.biodalliance.org/)
        - [BasePlayer](https://baseplayer.fi/)
        - [Biodalliance](http://www.biodalliance.org/)
        - [Celera genome browser](https://www.csee.umbc.edu/~turner/presentations/bvw2002/sld009.htm)
        - [Genoverse](https://github.com/wtsi-web/Genoverse)
        - [GBrowse 2](http://gmod.org/wiki/GBrowse)
        - [GenomeMaps](http://www.genomemaps.org/)
        - [Gosling](https://gosling.js.org/)
        - [GeneViTo](http://athina.biol.uoa.gr/bioinformatics/GENEVITO/)
        - [Ensembl genome browser](https://useast.ensembl.org/Homo_sapiens/Location/View?r=17:63992802-64038237)
        - [Ensembl genome browser 2020 edition](http://2020.ensembl.org/)
        - [HiGlass](https://higlass.io)
        - [IGB](https://bioviz.org/)
        - [IGV](https://igv.org/)
        - [IGV.js](https://github.com/igvteam/igv.js/)
        - [JBrowse](http://jbrowse.org/jbrowse1.html)
        - [JBrowse 2](http://jbrowse.org/jb2)
        - [Kero-BROWSE](https://kero.hgc.jp/examples/CLCL/hg38/index.html)
        - [NCBI Genome Data Viewer](https://www.ncbi.nlm.nih.gov/genome/gdv/)
        - [Nucleome browser](https://vis.nucleome.org/v1/main.html)
        - [Pileup.js](https://github.com/hammerlab/pileup.js)
        - [Savant](http://bioinformatics-ca.github.io/savant_genome_browser_lab_2015/)
        - [Tablet](https://ics.hutton.ac.uk/tablet/)
        - [Trackster](https://galaxyproject.org/learn/visualization/)
        - [UCSC genome browser](https://genome.ucsc.edu/)
        - [Valis browser](https://valis.bio/)
        - [WashU epigenomics browser](https://epigenomegateway.wustl.edu/)
        - [Zenbu](http://fantom.gsc.riken.jp/zenbu/)
    - Commercial
        - [10x genomics/Loupe](https://support.10xgenomics.com/genome-exome/software/visualization/latest/structural-variants)
        - [Alamut](https://www.interactive-biosoftware.com/alamut-visual/)
        - [ATCC Genome Portal (American Type Culture Collection)](https://docs.onecodex.com/en/articles/3996697-using-the-genome-browser)
        - [Bionano](https://bionanogenomics.com/technology/structural-variation/)
        - [Benchling](https://www.benchling.com/)
        - [CLC Genomics workbench](https://digitalinsights.qiagen.com/news/blog/discovery/structural-variant-detection-using-clc-genomics-workbench/)
        - [Circa](https://omgenomics.com/circa)
        - [DNASTAR](https://www.dnastar.com/)
        - [Geneious](https://www.geneious.com/)
        - [Genestack](https://genestack.com/blog/2015/05/28/navigation-in-genestack-genome-browser/)
        - [Golden Helix](http://goldenhelix.com/GenomeBrowse/)
        - [Lucid viewer](https://lucidalign.com/)
        - [MacVector](https://macvector.com/)
        - [Persephone](https://persephonesoft.com/)
        - [SnapGene](https://www.snapgene.com/)
        - [Strand NGS](https://www.strand-ngs.com/)
    - MSA
        - [AA](http://www.bioinformatics.org/strap/aa)
        - [abrowse](https://github.com/ihh/abrowse)
        - [alignment.js](https://github.com/veg/alignment.js)
        - [AlignmentComparator](http://bioinfweb.info/AlignmentComparator/)
        - [AlignmentViewer](https://alignmentviewer.org/)
        - [AliView](http://www.ormbunkar.se/aliview/)
        - [ALVIS (MSA viewer)](https://www.ebi.ac.uk/research/goldman/software/alvis)
        - [alignfigR](https://cran.r-project.org/web/packages/alignfigR/vignettes/my-vignette.html)
        - [alen](https://github.com/jakobnissen/alen)
        - [AliTreeViz](https://www.npmjs.com/package/alitreeviz)
        - [BioJS/MSA](https://msa.biojs.net/)
        - [Boxshade](https://embnet.vital-it.ch/software/BOX_form.html)
        - [ESPript](http://espript.ibcp.fr/ESPript/ESPript/)
        - [ggtree](https://github.com/YuLab-SMU/ggtree)
        - [Jalview](https://www.jalview.org/)
        - [JSAV](http://www.bioinf.org.uk/software/jsav)
        - [MSABrowser](https://thekaplanlab.github.io/)
        - [msaR](https://github.com/zachcp/msaR)
        - [MView](https://www.ebi.ac.uk/Tools/msa/mview/)
        - [MEGA-X](https://www.megasoftware.net/)
        - [NCBI MSA Viewer](https://www.ncbi.nlm.nih.gov/projects/msaviewer/)
        - [pyBoxshade](https://github.com/mdbaron42/pyBoxshade)
        - [pymsaplotter](https://github.com/orangeSi/pymsaploter)
        - [ProViz](http://slim.icr.ac.uk/proviz/index.php)
        - [React MSAViewer](https://github.com/plotly/react-msa-viewer)
        - [react-msaview](https://github.com/GMOD/react-msaview)
        - [seqotron](https://github.com/4ment/seqotron)
        - [STRAP](http://www.bioinformatics.org/strap/)
        - [SNPViz](http://www.yeastrc.org/snipviz/4.HTML_Config_retrieve_newick_and_fasta_from_server/snip_viz_HTML_config_with_newick_clustering.html)
        - [TeXShade](http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/texshade/texshade.pdf)
        - [Wasabi](http://wasabiapp.org/)
    
    - Logo
        - [LogoJS](https://logojs.wenglab.org/app/gallery/)
    
    - CNV
        - [aCNVViewer](https://github.com/FJD-CEPH/aCNViewer)
        - [BAMScale](https://github.com/ncbi/BAMscale)
        - [bctools cnv](https://samtools.github.io/bcftools/howtos/cnv-calling.html)
        - [CNVkit](https://cnvkit.readthedocs.io/en/stable/plots.html)
        - [cnvCurator](http://www.acsu.buffalo.edu/~lm69/cnvCurator)
        - [covviz](https://brwnj.github.io/covviz/)
        - [copynumber](https://www.bioconductor.org/packages/release/bioc/vignettes/copynumber/inst/doc/copynumber.pdf)
        - [GenomeSpy](https://genomespy.app/)
        - [GenomePaint](https://genomepaint.stjude.cloud/)
        - [genevisR](https://bioconductor.org/packages/release/bioc/vignettes/GenVisR/inst/doc/Intro.html)
        - [mocha](https://github.com/freeseek/mocha)
        - [PURPLE](https://github.com/hartwigmedical/hmftools/blob/master/purity-ploidy-estimator/README.md)
    - Static
        - [BAMSnap](https://github.com/parklab/bamsnap)
        - [bcftools roh](https://samtools.github.io/bcftools/howtos/roh-calling.html)
        - [ChIA-Pipe](https://github.com/TheJacksonLaboratory/ChIA-PIPE)
        - [CNView](https://github.com/RCollins13/CNView)
        - [CNVPlot](https://github.com/dantaki/CNVplot)
        - [CNVpytor](https://github.com/abyzovlab/CNVpytor)
        - [cgplot](https://github.com/dfguan/cgplot)
        - [DNAPlotLib](https://github.com/VoigtLab/dnaplotlib)
        - [ExonIntron](http://wormweb.org/exonintron)
        - [FeatureViewer](https://github.com/calipho-sib/feature-viewer)
        - [Geneviz](https://jrderuiter.github.io/geneviz/usage.html)
        - [Genome STRiP](http://software.broadinstitute.org/software/genomestrip/)
        - [GenomeTools](http://genometools.org/)
        - [GenomeView](https://github.com/nspies/genomeview)
        - [genoPlotR](http://genoplotr.r-forge.r-project.org/)
        - [ggbio](http://www.bioconductor.org/packages/2.11/bioc/vignettes/ggbio/inst/doc/ggbio.pdf)
        - [GGgenes](https://github.com/wilkox/gggenes)
        - [GGsashimi](https://github.com/guigolab/ggsashimi)
        - [gtrellis](http://bioconductor.org/packages/devel/bioc/vignettes/gtrellis/inst/doc/gtrellis.html)
        - [Gviz](https://bioconductor.org/packages/release/bioc/html/Gviz.html)
        - [Hagfish](https://github.com/mfiers/hagfish/wiki/Plots) 
        - [HiCPlotter](https://github.com/kcakdemir/HiCPlotter)
        - [JVarKit/BamToSVG](http://lindenb.github.io/jvarkit/BamToSVG.html)
        - [Lollipops](https://github.com/joiningdata/lollipops)
        - [Mason](http://www.yeastrc.org/mason/)
        - [Methplotlib](https://github.com/wdecoster/methplotlib)
        - [mummer2circos](https://github.com/metagenlab/mummer2circos)
        - [Pairoscope](http://pairoscope.sourceforge.net/)
        - [pauvre](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6991124/figure/fig-3/)
        - [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks)
        - [RACER](https://oliviasabik.github.io/RACERweb/articles/IntroToRACER.html)
        - [RepViz](https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-019-4473-z)
        - [Samplot](https://github.com/ryanlayer/samplot)
        - [SashimiPlot](https://miso.readthedocs.io/en/fastmiso/sashimi.html)
        - [shabam](https://github.com/dlrice/shabam)
        - [SparK](https://github.com/harbourlab/SparK)
        - [Spliceclust](https://github.com/pkimes/spliceclust)
        - [Sushi](https://www.bioconductor.org/packages/release/bioc/vignettes/Sushi/inst/doc/Sushi.pdf)
        - [svist4get](https://link.springer.com/article/10.1186/s12859-019-2706-8)
        - [svv](https://github.com/ryanlayer/svv)
        - [svviz](https://github.com/svviz/svviz)
        - [svviz2](https://github.com/nspies/svviz2)
        - [trackViewer](https://bioconductor.org/packages/release/bioc/vignettes/trackViewer/inst/doc/trackViewer.html)

## THINKS FOR 
- [awesome-genome-visualization](https://github.com/cmdcolin/)

The project welcomes any life science industry physician to add
        

